UITests/BraintreeDropIn_UITests.swift

```diff
@@ -955,6 +955,49 @@ class BraintreeDropIn_ThreeDSecure_2_UITests: XCTestCase {
     }
 }
 
+class BraintreeDropIn_ThreeDSecure_VaultedPaymentMethod_UITests: XCTestCase {
+    
+    var app: XCUIApplication!
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        app = XCUIApplication()
+        app.launchArguments.append("-EnvironmentSandbox")
+        app.launchArguments.append("-ClientToken")
+        app.launchArguments.append("-ThreeDSecureRequired")
+        app.launchArguments.append("-ThreeDSecureVersion2")
+        app.launchArguments.append("-EditModeCustomer")
+        app.launch()
+        sleep(1)
+        
+        waitForElementToBeHittable(app.buttons["Change Payment Method"])
+        app.buttons["Change Payment Method"].tap()
+    }
+    
+    func testDropIn_threeDSecure_2_challengeFlow_withVaultedPaymentMethod() {
+        waitForElementToBeHittable(app.collectionViews.cells.firstMatch)
```

Comment:
How do we know that the first vaulted payment method (if any) will be of the card payment type?

Reply:
It's a little bit fragile, but we're setting customer ID to `123-edit-test` in the set up method and that customer ID has the vaulted payment method we want. It's not ideal b/c deleting the payment method would break these tests, but we have other tests that are also relying on a vaulted payment method existing for that customer ID.

Reply:
Gotcha, thank you for clarifying! The name "-EditModeCustomer" indicates to me that the vault manager is enabled and that a customer can delete payment methods in their vault.

... but we're just using it for its vaulted cards.

Reply:
Yeah, that's true. Should we change it to `-CustomerWithVaultedCards` or something like that?

Reply:
So it seems to be currently used in[ other tests](https://github.com/braintree/braintree-ios-drop-in/blob/284eebbe4a9f0b6f970440311a73732dc283a1be/UITests/BraintreeDropinEditMode_UITests.swift#L17-L21) to see if `self.waitForElementToBeHittable(app.buttons["Change Payment Method"])`, or if vault manager is properly rendering.

So we could make it all encompassing to include the fact that 1) it has vaulted payments and 2) it shows edit mode.

`-CustomerEditVaultWithPayments`?

Reply:
It looks like the ability to edit vaulted payment methods is controlled by this flag: `-DisableEditMode`, not the customer ID. So I don't think we need to call out the edit mode stuff in the name of the customer ID flag.

Reply:
@scannillo I changed it to `-CustomerIDWithVaultedPaymentMethods`. Kinda wordy, but hopefully more descriptive? Lemme know if that works for you or if you have a better idea!